addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -63,6 +63,15 @@ class RecruitmentStage(models.Model):
     fold = fields.Boolean(
         "Folded in Recruitment Pipe",
         help="This stage is folded in the kanban view when there are no records in that stage to display.")
+    legend_blocked = fields.Char(
+        'Red Kanban Label', default=lambda s: _('Blocked'), translate=True, required=True,
+        help='Override the default value displayed for the blocked state for kanban selection, when the task or issue is in that stage.')
```

Comment:
Don't forget to change the help messages. IMHO, I don't even think it is necessary to have them. You can get rid of it.